test/celeritas/global/AlongStep.test.cc

```diff
@@ -228,7 +228,7 @@ TEST_F(MockAlongStepTest, basic)
         EXPECT_SOFT_EQ(1, result.angle);
         EXPECT_SOFT_NEAR(1.2431209185653e-12, result.time, 1e-11);
         EXPECT_SOFT_EQ(5.2704627669473e-05, result.step);
-        EXPECT_EQ("physics-discrete-select", result.action);
+        EXPECT_EQ("eloss-range", result.action);
```

Comment:
It looks like this changed because `celeriton` doesn't have an equivalent at-rest process. Can you add one so that we can check that we correctly test?